docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -0,0 +1,75 @@
+# macOS setup
+
+In Fleet you can customize the first time macOS setup experience for your end users.
+
+You can require end users to authenticate with your identity provider (IdP) and agree to and end user license agreement (EULA) before they can use their new Mac.
+
+You can customize the macOS Setup Assistant and choose to show or hide specific panes.
+
+Also, you can bootstrap new Macs with your configuration management tool of choice (ex. Munki, Chef, or Puppet).
+
+## End user authentication
+
+> This feature is currently in development.
+
+## Bootstrap package
+
+_Available in Fleet Premium_
+
+Fleet supports installing a single package on macOS hosts that automatically enroll to Fleet.
```

Comment:
I think we need to describe more about this feature. What does it do exactly? Why/when should I use it? Once you tell me that, then you tell me how to do it.

Reply:
I updated this section to describe what the bootstrap package, why you should use it (to install tools like Puppet Munki and custom scripts), and added some examples of what organizations deploy.

What do you think?